### PR TITLE
4.1.1.light bundle

### PR DESCRIPTION
**Pull request by tinazhang128**

### CODE REVIEW

File: aware-core/src/main/java/com/aware/Applications.java [hunk 1]
```diff
@@ -85,6 +90,55 @@ public class Applications extends AccessibilityService {
...
+    // Screentext variables
+    private String currScreenText = "";
+
+    ArrayList<ContentValues> contentBuffer = new ArrayList<ContentValues>();
+
+    ArrayList<Integer> textBuffer = new ArrayList<Integer>();
+
+    int TEXT_BUFFER_LIMIT = 100;
+
+    int mDebugDepth = 0;
```
Comment (Songyan12345) [inline aware-core/src/main/java/com/aware/Applications.java:102]: Does this still need to be included?

**Review by Songyan12345 — CHANGES_REQUESTED** [review]
Added comments on storing the debug depth and removing the screen text that does not include positions

File: aware-core/src/main/java/com/aware/Applications.java (hunk 1 above)
Comment (tinazhang128) [reply aware-core/src/main/java/com/aware/Applications.java:102]: Do you mean TEXT_BUFFER_LIMIT?

Comment (Songyan12345) [reply aware-core/src/main/java/com/aware/Applications.java:102]: Sorry, mDebugDepth

File: aware-core/src/main/java/com/aware/Applications.java [hunk 2]
```diff
@@ -85,6 +90,55 @@ public class Applications extends AccessibilityService {
...
+     * Recursively track all the text on the screen in a tree structure to the text variable
+     *
+     * @param mNodeInfo
+     */
+    private void textTree(AccessibilityNodeInfo mNodeInfo){
+        if (mNodeInfo == null) return;
+
+        // conditions to filter the meaningless input
+        if (mNodeInfo.getText() != null && !mNodeInfo.getText().toString().equals("")){
+            currScreenText += mNodeInfo.getText() + "||"; // Add division sign for the tree
```
[the remark this answers is not in the document]
Comment (tinazhang128) [reply aware-core/src/main/java/com/aware/Applications.java:125]: Do you mean line 125 can be discarded?

File: aware-core/src/main/java/com/aware/Applications.java (hunk 1 above)
Comment (tinazhang128) [reply aware-core/src/main/java/com/aware/Applications.java:102]: Well, it seems to be used when we follow the tree structure of the widgets on the screen.

File: aware-core/src/main/java/com/aware/Applications.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (tinazhang128) [reply aware-core/src/main/java/com/aware/Applications.java:125]: Sure, I will do that soon.

**Review by Songyan12345 — APPROVED** [review]
LGMT, thank you very much! :)

**Pull request merged**